src/convertToObject.js

```diff
@@ -3,10 +1,30 @@
 /**
- * @param {string} sourceString
+ * @param {string} stylesString
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const result = {};
+
+  // Разделение на ключ-значение
+  stylesString.split(';').forEach((line) => {
+    // Редактируем строку
+    const cleanedLine = line.trim();
+
+    // Пропускаем пробелы
+    if (!cleanedLine) {
+      return;
+    }
+
+    const [property, value] = cleanedLine.split(':').map((part) => part.trim());
```

Comment:
The use of `map((part) => part.trim())` is correct for trimming spaces from both the property and value. Ensure that the input string is well-formed to avoid unexpected results.